JustSaying/IMessagingBus.cs

```diff
@@ -0,0 +1,16 @@
+using System.Threading;
+
+namespace JustSaying
+{
+    /// <summary>
+    /// Defines a messaging bus.
+    /// </summary>
+    public interface IMessagingBus
```

Comment:
When you get around to publishers, I suggest renaming `IMessagingBus` to something that indicates it's subscriber specific

Reply:
Not sure what you mean by this?

Reply:
This interface has a `Start` method, which means nothing if you are just publishing

Reply:
Ah, okay. Well if we do as @adammorr and I suggested above where we make the fluent API branch into publisher/subscriber config, then this `Start` method would disappear.